.codecov.yml

```diff
@@ -12,37 +12,5 @@ coverage:
 #     generator/generator_config.textproto | sort -u
```

Comment:
should we modify this comment block now?

```
# Ignore generated code that lives with handcrafted code.
```

Reply:
Done.